### DISCUSSION TITLE
Unable to add member in The Group

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#482] [Sunbird ED configuration discussion] [answered]
Hello All,

I'm using 4.6 version of mobile app.

Adding member to group was working previously. But suddenly it stopped.

**_Issue came to know:_**
On debugging, got to know that _https://sunbirdsaas.com/api/data/v1/system/settings/get/appGoogleReCaptcha_ is getting **404 error**

**_Solution Tried:_**
Added packages and created captcha key. 
Now we have another error in the learner logs. Sharing the logs below.

`learner-74b5fc95bd-flq48 {"timestamp":"2023-05-09T09:16:47.577Z","msg":"{\"eid\":\"LOG\",\"ets\":1683623807577,\"ver\":\"3.0\",\"mid\":\"fd92629754ebaac70b3f265dfdca95cf\",\"actor\":{\"id\":\"bd774f1a-05fe-4e9d-
93d8-87b8387fbbf3\",\"type\":\"Consumer\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"Miscellaneous\",\"did\":\"5
731a0982a1b6404af1750194eda00f8b13c3ef7\",\"cdata\":[{\"id\":\"fd92629754ebaac70b3f265dfdca95cf\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"level\":\"info\",\"type\":\"Api_access\",\"message\":\"{eid='L
OG', edata={level=trace, requestid=fd92629754ebaac70b3f265dfdca95cf, type=system, message=EXIT LOG: method : GET, url: /v1/system/settings/get/hotCodePush , For Operation : getSystemSetting, params=[{msgid=fd926
29754ebaac70b3f265dfdca95cf, errmsg=Requested resource not found, resmsgid=null, err=RESOURCE_NOT_FOUND, status=FAILED, responseCode=404}]}}\",\"params\":[]}}","lname":"TelemetryEventLogger","tname":"Thread-18",
"level":"INFO","HOSTNAME":"learner-74b5fc95bd-flq48","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}`

Can anyone help?

cc: @swayangjit

**Comment by swayangjit** [comment]
@nidhikachhawa24  Have you configured appGoogleReCaptcha in System settings ?

**Reply by nidhikachhawa24** [reply]
@swayangjit 
Yes we've configured appGoogleReCaptacha in System settings

cc: @raghupathiguduri

**Reply by swayangjit** [reply]
@nidhikachhawa24  Please send the response from get call of system settings API.

**Reply by nidhikachhawa24** [reply]
@raghupathiguduri please share the response from get call of system settings API

**Reply by swayangjit** [reply]
@nidhikachhawa24  Any update on this ?

**Reply by raghupathiguduri** [reply]
@swayangjit We have created key in google recaptcha enterprise by adding the packegs. 
Next we have updated the systemsettings to get appGoogleReCaptcha with secret key and site key in the learner configmap varibale `google_captcha_mobile_private_key`. 

response from get call of system settings API
{"id":"api.system.settings.get.appGoogleReCaptcha","ver":"v1","ts":"2023-05-16 10:57:20:541+0000","params":{"resmsgid":null,"msgid":"d86d40298fa6db13241ef3ca3b4eedbc","err":null,"status":"success","errmsg":null},"responseCode":"OK","result":{"response":{"id":"appGoogleReCaptcha","field":"appGoogleReCaptcha","value":"{\"key\":\"6LfmJxImAAAAAHXnBYr5hzHtd1CBkwt-LXMlyLrD\" , \"isEnabled\":true}"}}}

But still issue didn't resolved. Below is the error message coming in the mobile app display. 
`Due to a technical problem we are unable to process the request`

And below are the error logs in the learner service.


learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:25.351825522+05:30 {"timestamp":"2023-05-16T12:42:25.351Z","msg":"{\"eid\":\"LOG\",\"ets\":1684240945351,\"ver\":\"3.0\",\"mid\":\"fb4534324c7bba9683316c4e11c908eb\",\"actor\":{\"id\":\"bd774f1a-05fe-4e9d-93d8-87b8387fbbf3\",\"type\":\"Consumer\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"Miscellaneous\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"fb4534324c7bba9683316c4e11c908eb\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"level\":\"info\",\"type\":\"Api_access\",\"message\":\"{eid='LOG', edata={level=trace, requestid=fb4534324c7bba9683316c4e11c908eb, type=system, message=ENTRY LOG: method : GET, url: /v1/system/settings/get/appGoogleReCaptcha , For Operation : getSystemSetting, params=[{field=appGoogleReCaptcha, id=null, userId=null}]}}\",\"params\":[]}}","lname":"TelemetryEventLogger","tname":"application-akka.actor.default-dispatcher-7","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:25.352878299+05:30 {"timestamp":"2023-05-16T12:42:25.352Z","msg":"{\"eid\":\"LOG\",\"ets\":1684240945352,\"ver\":\"3.0\",\"mid\":\"fb4534324c7bba9683316c4e11c908eb\",\"actor\":{\"id\":\"bd774f1a-05fe-4e9d-93d8-87b8387fbbf3\",\"type\":\"Consumer\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"Miscellaneous\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"fb4534324c7bba9683316c4e11c908eb\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"level\":\"info\",\"type\":\"Api_access\",\"message\":\"\",\"params\":[{\"method\":\"GET\"},{\"url\":\"/v1/system/settings/get/appGoogleReCaptcha\"},{\"duration\":0},{\"status\":\"OK\"}]}}","lname":"TelemetryEventLogger","tname":"Thread-478","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:25.353340658+05:30 {"timestamp":"2023-05-16T12:42:25.353Z","msg":"{\"eid\":\"LOG\",\"ets\":1684240945353,\"ver\":\"3.0\",\"mid\":\"fb4534324c7bba9683316c4e11c908eb\",\"actor\":{\"id\":\"bd774f1a-05fe-4e9d-93d8-87b8387fbbf3\",\"type\":\"Consumer\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"Miscellaneous\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"fb4534324c7bba9683316c4e11c908eb\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"level\":\"info\",\"type\":\"Api_access\",\"message\":\"{eid='LOG', edata={level=trace, requestid=fb4534324c7bba9683316c4e11c908eb, type=system, message=EXIT LOG: method : GET, url: /v1/system/settings/get/appGoogleReCaptcha , For Operation : getSystemSetting, params=[{response=org.sunbird.model.systemsettings.SystemSetting@4ec0a044}, {msgid=fb4534324c7bba9683316c4e11c908eb, errmsg=null, resmsgid=null, err=null, status=success, responseCode=200}]}}\",\"params\":[]}}","lname":"TelemetryEventLogger","tname":"Thread-478","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.722669098+05:30 {"timestamp":"2023-05-16T12:42:27.721Z","msg":"Perhaps this is another API, like search that doesn't carry user id.","lname":"util.RequestInterceptor","tname":"application-akka.actor.default-dispatcher-133","level":"ERROR","stack_trace":"<#d303254c> java.lang.IllegalArgumentException: Invalid UUID string: Support\n\tat java.util.UUID.fromString(UUID.java:215)\n\tat util.RequestInterceptor.getUserRequestedFor(RequestInterceptor.java:106)\n\tat util.RequestInterceptor.verifyRequestData(RequestInterceptor.java:138)\n\tat modules.OnRequestHandler$1.call(OnRequestHandler.java:60)\n\tat play.core.j.JavaAction$$anonfun$10.apply(JavaAction.scala:191)\n\tat play.core.j.JavaAction$$anonfun$10.apply(JavaAction.scala:191)\n\tat scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)\n\tat scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)\n\tat play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:79)\n\tat play.api.libs.streams.Execution$trampoline$.executeScheduled(Execution.scala:111)\n\tat play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:73)\n\tat scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:44)\n\tat scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:252)\n\tat scala.concurrent.Promise$class.complete(Promise.scala:55)\n\tat scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:157)\n\tat scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23)\n\tat akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)\n\tat akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:44)\n\tat akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)\n\tat akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)\n\tat akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)\n\tat akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)\n","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.723082678+05:30 {"timestamp":"2023-05-16T12:42:27.722Z","msg":"{\"eid\":\"LOG\",\"ets\":1684240947722,\"ver\":\"3.0\",\"mid\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"actor\":{\"id\":\"33269762-8a19-485d-8caf-123b36bbd111\",\"type\":\"User\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"User\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"level\":\"info\",\"type\":\"Api_access\",\"message\":\"{eid='LOG', edata={level=trace, requestid=a8684c204f862bd0d4efd4b4ff90cbad, type=system, message=ENTRY LOG: method : GET, url: /v2/user/exists/userName/Support , For Operation : checkUserExistenceV2, params=[{id=null, userId=null}]}}\",\"params\":[]}}","lname":"TelemetryEventLogger","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.723101932+05:30 {"timestamp":"2023-05-16T12:42:27.722Z","msg":"Calling Api: https://www.google.com/recaptcha/api/siteverify?response&secret=6LekHPEiAAAAALhRuaY3sZTDFN6UzecVyems3eLn","lname":"util.CaptchaHelper","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.723107322+05:30 {"timestamp":"2023-05-16T12:42:27.722Z","msg":"Captcha: null","lname":"util.CaptchaHelper","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.796110625+05:30 {"timestamp":"2023-05-16T12:42:27.795Z","msg":"Got response from postFormData call : {\n  \"success\": false,\n  \"error-codes\": [\n    \"invalid-input-response\"\n  ]\n}","lname":"org.sunbird.http.HttpClientUtil","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.796396692+05:30 {"timestamp":"2023-05-16T12:42:27.796Z","msg":"exception in validating the google captcha: [invalid-input-response]","lname":"util.CaptchaHelper","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.796722068+05:30 {"timestamp":"2023-05-16T12:42:27.796Z","msg":"{\"eid\":\"ERROR\",\"ets\":1684240947796,\"ver\":\"3.0\",\"mid\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"actor\":{\"id\":\"33269762-8a19-485d-8caf-123b36bbd111\",\"type\":\"User\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"User\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"err\":\"INVALID_CAPTCHA\",\"stacktrace\":\"BaseController:handleRequest for operation: checkUserExistenceV2 Exception occurred with error message = Captcha is invalid controllers.usermanagement.validator.UserGetRequestValidator.validateGetUserByKeyRequestaWithCaptcha(UserGetRequestValidator.java:64)controllers.usermanagement.UserController.lambda$userExists$18(UserController.java:450)controllers.BaseController.handleRequest(BaseController.java:262)controllers.BaseController.handleRequest(BaseController.java:227)controllers.usermanagement.UserController.userExists(UserController.java:442)router.Routes$$anonfun$routes$1$$anonfun$applyOrElse$23$$anonfun$apply$128$$anonfun$apply$129.apply(Routes.scala:2034)router.Routes$$anonfun$routes$1$$anonfun$applyOrElse$23$$anonfun$apply$128$$anonfun$apply$129.apply(Routes.scala:2034)play.core.routing.HandlerInvokerFactory$$anon$6.resultCall(HandlerInvoker.scala:155)play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$10$$anon$2$$anon$1.invocation(HandlerInvoker.scala:116)play.core.j.JavaAction$$anon$1.call(JavaAction.scala:131)play.mvc.Action.lambda$call$0(Action.java:89)java.base/java.util.Optional.map(Optional.java:265)play.mvc.Action.call(Action.java:81)modules.OnRequestHandler$1.call(OnRequestHandler.java:83)play.core.j.JavaAction$$anonfun$10.apply(JavaAction.scala:191)play.core.j.JavaAction$$anonfun$10.apply(JavaAction.scala:191)scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:79)play.api.libs.streams.Execution$trampoline$.executeScheduled(Execution.scala:111)play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:73)scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:44)scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:252)scala.concurrent.Promise$class.complete(Promise.scala:55)scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:157)scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23)akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:44)akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)akka.dispatch.forkj\",\"errtype\":\"api_access\",\"requestid\":\"a8684c204f862bd0d4efd4b4ff90cbad\"}}","lname":"TelemetryEventLogger","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.797279326+05:30 {"timestamp":"2023-05-16T12:42:27.796Z","msg":"{\"eid\":\"LOG\",\"ets\":1684240947796,\"ver\":\"3.0\",\"mid\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"actor\":{\"id\":\"33269762-8a19-485d-8caf-123b36bbd111\",\"type\":\"User\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"User\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"level\":\"info\",\"type\":\"Api_access\",\"message\":\"{eid='LOG', edata={level=trace, requestid=a8684c204f862bd0d4efd4b4ff90cbad, type=system, message=EXIT LOG: method : GET, url: /v2/user/exists/userName/Support , For Operation : checkUserExistenceV2, params=[{msgid=a8684c204f862bd0d4efd4b4ff90cbad, errmsg=Captcha is invalid, resmsgid=null, err=INVALID_CAPTCHA, status=FAILED, responseCode=418}]}}\",\"params\":[]}}","lname":"TelemetryEventLogger","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.797305115+05:30 {"timestamp":"2023-05-16T12:42:27.796Z","msg":"{\"eid\":\"ERROR\",\"ets\":1684240947796,\"ver\":\"3.0\",\"mid\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"actor\":{\"id\":\"33269762-8a19-485d-8caf-123b36bbd111\",\"type\":\"User\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"User\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"err\":\"418\",\"stacktrace\":\"controllers.usermanagement.validator.UserGetRequestValidator.validateGetUserByKeyRequestaWithCaptcha\",\"errtype\":\"INVALID_CAPTCHA\",\"requestid\":\"a8684c204f862bd0d4efd4b4ff90cbad\"}}","lname":"TelemetryEventLogger","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}
learner-56fb5c679-4j9z5 learner 2023-05-16T18:12:27.797320530+05:30 {"timestamp":"2023-05-16T12:42:27.797Z","msg":"{\"eid\":\"ERROR\",\"ets\":1684240947797,\"ver\":\"3.0\",\"mid\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"actor\":{\"id\":\"33269762-8a19-485d-8caf-123b36bbd111\",\"type\":\"User\"},\"context\":{\"channel\":\"01373263149320601644\",\"pdata\":{\"id\":\"org.key.app\",\"pid\":\"learner-service\",\"ver\":\"4.6.0\"},\"env\":\"User\",\"did\":\"a0d04c6bdcec96172826fe36c8535737547a1823\",\"cdata\":[{\"id\":\"a8684c204f862bd0d4efd4b4ff90cbad\",\"type\":\"Request\"}],\"rollup\":{}},\"edata\":{\"err\":\"INVALID_CAPTCHA\",\"stacktrace\":\"Captcha is invalid controllers.usermanagement.validator.UserGetRequestValidator.validateGetUserByKeyRequestaWithCaptcha(UserGetRequestValidator.java:64)controllers.usermanagement.UserController.lambda$userExists$18(UserController.java:450)controllers.BaseController.handleRequest(BaseController.java:262)controllers.BaseController.handleRequest(BaseController.java:227)controllers.usermanagement.UserController.userExists(UserController.java:442)router.Routes$$anonfun$routes$1$$anonfun$applyOrElse$23$$anonfun$apply$128$$anonfun$apply$129.apply(Routes.scala:2034)router.Routes$$anonfun$routes$1$$anonfun$applyOrElse$23$$anonfun$apply$128$$anonfun$apply$129.apply(Routes.scala:2034)play.core.routing.HandlerInvokerFactory$$anon$6.resultCall(HandlerInvoker.scala:155)play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$10$$anon$2$$anon$1.invocation(HandlerInvoker.scala:116)play.core.j.JavaAction$$anon$1.call(JavaAction.scala:131)play.mvc.Action.lambda$call$0(Action.java:89)java.base/java.util.Optional.map(Optional.java:265)play.mvc.Action.call(Action.java:81)modules.OnRequestHandler$1.call(OnRequestHandler.java:83)play.core.j.JavaAction$$anonfun$10.apply(JavaAction.scala:191)play.core.j.JavaAction$$anonfun$10.apply(JavaAction.scala:191)scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:79)play.api.libs.streams.Execution$trampoline$.executeScheduled(Execution.scala:111)play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:73)scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:44)scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:252)scala.concurrent.Promise$class.complete(Promise.scala:55)scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:157)scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23)akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:44)akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)\",\"errtype\":\"api_access\",\"requestid\":\"a8684c204f862bd0d4efd4b4ff90cbad\"}}","lname":"TelemetryEventLogger","tname":"application-akka.actor.default-dispatcher-133","level":"INFO","HOSTNAME":"learner-56fb5c679-4j9z5","application.home":"/home/sunbird/learner/learning-service-1.0-SNAPSHOT"}

**Reply by swayangjit** [reply]
@nidhikachhawa24  Created a [lern  discussion thread](https://github.com/orgs/Sunbird-Lern/discussions/119) for the same

**Reply by swayangjit** [reply, marked as answer]
@nidhikachhawa24 The issue was the site-key was created using the reCaptcha enterprise edition but we are supporting recaptcha classic. The adopter has created the site-key using **reCaptcha classic** and it's working now.
